crates/ruff_python_formatter/src/comments/placement.rs

```diff
@@ -465,29 +469,135 @@ fn handle_positional_only_arguments_separator_comment<'a>(
     }
 }
 
-fn find_pos_only_slash_offset(trivia_range: TextRange, locator: &Locator) -> Option<TextSize> {
-    let mut in_comment = false;
+/// Handles comments between the left side and the operator of a binary expression (trailing comments of the left),
+/// and trailing end-of-line comments that are on the same line as the operator.
+///
+///```python
+/// a = (
+///     5 # trailing left comment
+///     + # trailing operator comment
```

Comment:
Where should this be placed, in the formatted output?

Reply:
The same as shown in this code snippet at the end of the operator line.